updates to different articles chapter (#2409)
alan-turing-institute/the-turing-way: book/website/reproducible-research/rdm/rdm-article.md

It also facilitates recognition for datasets through research assessment procedures that are more traditionally focused on publication output.
Publishing a data paper will therefore increase the visibility, credibility and usability of the data, as well as giving you credit as a data producer.

> You can find out more about the benefits of publishing a data article through [this subchapter](https://the-turing-way.netlify.app/communication/dif-articles/data.html).

Reviewer: ```suggestion
You can find out more in our {ref}`subchapter on publishing a data article<cm-dif-articles-data>`.
```
